core/templates/Polyfills.ts

```diff
@@ -159,3 +159,17 @@ if (navigator.mediaDevices.getUserMedia === undefined) {
     });
   };
 }
+
+// Create a polyfill for Object.entries() for Chrome.
```

Comment:
Please specify the range of target Chrome versions. Also please rephrase the comment such that it sounds like a description, not a task.

Reply:
Done